src/bid.rs

```diff
@@ -6,11 +6,15 @@
 
 //! Bid data structure
 
+pub(crate) mod encoding;
+pub(crate) mod score;
 use crate::errors::BlindBidError;
-#[cfg(feature = "canon")]
-use canonical::Canon;
-#[cfg(feature = "canon")]
-use canonical_derive::Canon;
+cfg_if::cfg_if! {
+    if #[cfg(feature = "canon")] {
+        use canonical::Canon;use canonical_derive::Canon;
+    }
+}
```

Comment:
For just two line I don't think is worthy, it's probably more concise use the previous form – I suggested `cfg-if` only for the massive "std" in score.

Also, the formatting is a bit weird, the two `use` should be on two different lines.

Reply:
Perfect